src/app/modules/rates/sagas.ts

```diff
@@ -8,9 +8,9 @@ import types from './types';
 export function* handleFetchRates(): SagaIterator {
   try {
     const fiats = Object.keys(Fiat);
-    const btcRate = yield call(apiFetchRates, 'BTC', fiats);
-    const ltcRate = yield call(apiFetchRates, 'LTC', fiats);
-    const rates = { btcRate, ltcRate }
+    const bitcoin = yield call(apiFetchRates, 'BTC', fiats);
+    const litecoin = yield call(apiFetchRates, 'LTC', fiats);
+    const rates = { bitcoin, litecoin };
```

Comment:
It would be great to make this programmatic so that new CHAIN_TYPEs have their rates fetched, but I'd be happy to make that change myself.

Reply:
Ah yes, I forgot about this suggestion. Will do!